prospector/tools/pylint/__init__.py

```diff
@@ -62,8 +62,6 @@ def _prospector_configure(self, prospector_config, linter):
         # was disabled.
         linter.disable(
             'locally-disabled')  # notification about disabling a message
-        linter.disable(
```

Comment:
I'm not sure about a direct removal of the message, perhaps checking the version prior to that would be better. What's your take on this @carlio ?